### DISCUSSION TITLE
Feedback: comments in dark mode are poorly legligable

### DISCUSSION THREAD

**Discussion by matklad** [community/community#23935] [Accessibility]
<p>I’ve used dark theme on GitHub for several weeks, but than switched back to a light one. The reason for that is that I spend a lot of time writing and reading comments, and, in the dark theme, this is just hard on the eyes. I think part of the problem is <code>14</code> font size, which is just too small, but there might be some other typography tricks.</p>
<p>Note that I am a long-time user of dark themes in Discoures and Gmail, and they don’t suffer from this problem. Here are a couple of screenshots:</p>
<p>Gmail dark:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/c/8c558bd33423b7073f359a01d89df0af067116fa.png" data-download-href="/uploads/short-url/k1s7V0iB8sVAwEFeg1eLYFcHOmm.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1711539/181080290-bea74e6e-860b-47f6-9a70-a986b094a633.png" alt="image" data-base62-sha1="k1s7V0iB8sVAwEFeg1eLYFcHOmm" width="690" height="282" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c558bd33423b7073f359a01d89df0af067116fa_2_690x282.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c558bd33423b7073f359a01d89df0af067116fa_2_1035x423.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c558bd33423b7073f359a01d89df0af067116fa_2_1380x564.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c558bd33423b7073f359a01d89df0af067116fa_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2193×898 84.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Discourse dark:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/f/dfe8637098b94a64dbb0873ddba23b59f19d3d58.png" data-download-href="/uploads/short-url/vWMmGhS5Fxa9XvJHFLVTWpdYdhS.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1711539/181080292-1453abcf-15ea-47e5-ab82-0a8846bbb374.png" alt="image" data-base62-sha1="vWMmGhS5Fxa9XvJHFLVTWpdYdhS" width="689" height="288" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/f/dfe8637098b94a64dbb0873ddba23b59f19d3d58_2_689x288.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/f/dfe8637098b94a64dbb0873ddba23b59f19d3d58_2_1033x432.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/f/dfe8637098b94a64dbb0873ddba23b59f19d3d58_2_1378x576.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/f/dfe8637098b94a64dbb0873ddba23b59f19d3d58_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2446×1023 307 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>GitHub dark:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/3/03cc06a5e390c9f7d657ac4ade407044626f84b6.png" data-download-href="/uploads/short-url/xAytokOCqOeP72CAgsL7VuKBLM.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1711539/181080293-33b16ddf-8b8a-438c-86be-d4a838429367.png" alt="image" data-base62-sha1="xAytokOCqOeP72CAgsL7VuKBLM" width="690" height="213" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/3/03cc06a5e390c9f7d657ac4ade407044626f84b6_2_690x213.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/3/03cc06a5e390c9f7d657ac4ade407044626f84b6_2_1035x319.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/3/03cc06a5e390c9f7d657ac4ade407044626f84b6_2_1380x426.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/3/03cc06a5e390c9f7d657ac4ade407044626f84b6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1595×493 79.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ThreePalmTrees** [comment]
<p>I highly suggest using Amino extension for chrome,<br>
You can override any css rule including font size and background color</p>

**Comment by Andre601** [comment]
<p>I for my part don’t experience those issues, so it’s probably down to personal preferences and experience.</p>
<p>The examples you showed all have a thing in common: They either have a light text box (GMail) or a brighter grey background (Discourse).</p>
<p>I personally like the current setup of GitHub’s dark theme and as mentioned before don’t suffer the same issues as you do, so maybe it’s also down to your PC/Screen setup and eye conditions?</p>

**Comment by matklad** [comment, marked as answer]
<p>The new “dark dimmed theme” solves the problem for me:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/b/7b24971d3d88f752e2793b407203e5dc2fb6fb2b.png" data-download-href="/uploads/short-url/hzn7DzYg9A6oBtC2GKL5x3RDAcX.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1711539/181080298-6e5dedf5-a56e-44ae-8225-4f670a6b53c5.png" alt="image" data-base62-sha1="hzn7DzYg9A6oBtC2GKL5x3RDAcX" width="690" height="379" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b24971d3d88f752e2793b407203e5dc2fb6fb2b_2_690x379.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b24971d3d88f752e2793b407203e5dc2fb6fb2b_2_1035x568.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/b/7b24971d3d88f752e2793b407203e5dc2fb6fb2b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b24971d3d88f752e2793b407203e5dc2fb6fb2b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1172×645 36 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by cehfisher** [comment]
Hi @matklad @queenofcorgis - this issue is resolved [per this comment from the original poster](https://github.com/orgs/community/discussions/23935#discussioncomment-3242267). Could you please close this issue? Thanks!
